internal/server/promMetrics/metrcis.go

```diff
@@ -59,12 +59,17 @@ func SetRuleStatusCountGauge(isRunning bool, count int) {
 	RuleStatusCountGauge.WithLabelValues(lbl).Set(float64(count))
 }
 
-func SetRuleStatus(ruleID string, isRunning bool) {
-	v := 0
-	if isRunning {
-		v = 1
-	}
-	RuleStatusGauge.WithLabelValues(ruleID).Set(float64(v))
+type RuleStatusMetricsValue int
+
+const (
```

Comment:
Can we move this to another package? It does not make sense for server to refer to this package, especially prometheus can be turned off.